pandas/core/tools/datetimes.py

```diff
@@ -902,7 +902,7 @@ def to_datetime(
     in addition to forcing non-dates (or non-parseable dates) to :const:`NaT`.
 
     >>> pd.to_datetime("13000101", format="%Y%m%d", errors="coerce")
-    NaT
+    Timestamp('1300-01-01 00:00:00')
```

Comment:
Nice!

Reply:
definitely better behavior, but no longer matches what the text above this says it is demonstrating.  will update the example.